tests/app/letters/test_letter_utils.py

```diff
@@ -200,6 +200,7 @@ def test_get_letter_pdf_gets_pdf_from_correct_bucket(
     assert ret == b'pdf_content'
 
 
+@freeze_time("2020-12-11 17:31:00")
```

Comment:
Builds on the 3rd commit failed because of a flaky test: a filename is generated based on the current date and the second was not the same during the generation and the assertion. Made a commit to fix the flaky test.

Test output was (you can see it in the logs)
```
E       AssertionError: Expected call: s3upload(bucket_name='test-letters-pdf', file_location='2020-12-14/NOTIFY.FOO.D.2.C.C.20201214094655.PDF', filedata=b'\x00\x01', region='us-east-1')
E       Actual call: s3upload(bucket_name='test-letters-pdf', file_location='2020-12-14/NOTIFY.FOO.D.2.C.C.20201214094656.PDF', filedata=b'\x00\x01', region='us-east-1')
E       
E       pytest introspection follows:
E       
E       Kwargs:
E       assert {'bucket_name...: 'us-east-1'} == {'bucket_name'...: 'us-east-1'}
E         Common items:
E         {'bucket_name': 'test-letters-pdf',
E          'filedata': b'\x00\x01',
E          'region': 'us-east-1'}
E         Differing items:
E         {'file_location': '2020-12-14/NOTIFY.FOO.D.2.C.C.20201214094656.PDF'} != {'file_location': '2020-12-14/NOTIFY.FOO.D.2.C.C.20201214094655.PDF'}
E         Full diff:
E         {'bucket_name': 'test-letters-pdf',
E         -  'file_location': '2020-12-14/NOTIFY.FOO.D.2.C.C.20201214094656.PDF',
E         ?                                                               ^
E         +  'file_location': '2020-12-14/NOTIFY.FOO.D.2.C.C.20201214094655.PDF',
E         ?                                                               ^
E         'filedata': b'\x00\x01',
E         'region': 'us-east-1'}
```